pages/news/_id.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <v-container>
+    <v-row
+    >
+      <v-col
+        cols="12"
+        md="9"
+      >
+        <SpecificNewsCard
+          :id="this.$route.params.id"
+        />
+      </v-col>
+      <v-col
+        cols="12"
+        md="3"
+      >
+        <GreyCard />
+      </v-col>
+    </v-row>
+    <OtherNews class="mt-10 mx-0 mb-10" />
```

Comment:
`my` sets both `mt` and `mb`
```suggestion
    <OtherNews class="my-10 mx-0" />
```